test/api.js

```diff
@@ -73,8 +73,6 @@ const main = async ({ database, generateApp }, args) => {
         process.exit(1);
       });
     });
```

Comment:
Is it completely necessary to stdout that Tests failed? Jest logs will log it failed. So maybe the catch could directly call the process.exit(1)

Reply:
I was wondering I think I can remove this